pages/emailSender.tsx

```diff
@@ -75,6 +75,25 @@ const EmailSender: NextPage = () => {
       : setSubjectCustomization(true)
   }
 
+  const parentCallback = (id: string, to: string, subject: string, messageContent : string) => {
```

Comment:
Let's change this function name to be more descriptive of what it's doing in our application - maybe editFinalMessages or another name you think would be good!